crates/proof-of-sql/src/sql/proof_plans/filter_exec.rs

```diff
@@ -152,14 +152,16 @@ impl ProverEvaluate for FilterExec {
     #[tracing::instrument(name = "FilterExec::result_evaluate", level = "debug", skip_all)]
     fn result_evaluate<'a, S: Scalar>(
         &self,
-        input_length: usize,
         alloc: &'a Bump,
         accessor: &'a dyn DataAccessor<S>,
     ) -> Vec<Column<'a, S>> {
+        let input_length = accessor.get_length(self.table.table_ref);
         // 1. selection
-        let selection_column: Column<'a, S> =
-            self.where_clause
-                .result_evaluate(input_length, alloc, accessor);
+        let selection_column: Column<'a, S> = self
+            .where_clause
+            .result_evaluate(alloc, accessor)
+            .into_column(input_length, alloc)
+            .expect("Failed to convert columnar value to column");
```

Comment:
Can you add a `# Panics` doc to `ProverEvaluate` indicating that the underlying columns must all have the same length?